src/shared/constants.js

```diff
@@ -0,0 +1,9 @@
+export const PARAM_VIEW_MODE = 'view_mode';
```

Comment:
i heartily endorse this methodology for constants.

do not export default an object. this pointlessly removes es6 module magic.

you can really just write
```export const FOO = 'bar';```
it's ok, i promise